README.md

```diff
@@ -129,6 +145,8 @@ Criteria:
 		- User Story
 	Start Date: 2001-01-23
 	End Date: 2019-12-30
+	BlockedAttributes: Blocked, Blocked-In Progress
```

Comment:
Are "blocked" and "blocked-in progress" workflow states, or is this looking at the "Flagged" attribute in JIRA?

Reply:
So, the way this works is that it will evaluate each field in the change log and if it matches any of the entries in the BlockedAttributes criteria, it will calculate the number of blocked days based upon the on/off of that field value.  I believe that if you wanted to calculate the blocked days based upon the "Flagged" condition, you would just put the field "Impediment" in the BlockedAttributes section.  Like this:  BlockedAttributes: Impediment

Reply:
Understood, unfortunately, that is not working for me. I tried adding Impediment, Flagged, Blocked, and none worked. There is no Blocked Days column being added to the output file.

Reply:
okay, I'll see what I can do.  I'll test specifically with Impediment and Flagged.